cmake/modules/FindTENSORRT.cmake

```diff
@@ -9,22 +9,24 @@ endif()
 
 find_path(
     TENSORRT_INCLUDE_DIR NvInfer.h
-    HINTS ${TENSORRT_DIR}
+    HINTS ${TENSORRT_DIR} ${CUDA_TOOLKIT_ROOT_DIR}
     PATH_SUFFIXES include)
 
 if (NOT TENSORRT_INCLUDE_DIR)
-    message(FATAL_ERROR "Cannot find TensorRT header NvInfer.h, "
-        "please check if the path is correct")
+    message(FATAL_ERROR "Cannot find TensorRT header NvInfer.h "
+        "in TENSORRT_DIR: ${TENSORRT_DIR} or in CUDA_TOOLKIT_ROOT_DIR: "
```

Comment:
Why do we search TensorRT in `CUDA_TOOLKIT_ROOT_DIR`?

Reply:
It was brought from master to dev-0.5.0